pages/Plugins 101.md

```diff
@@ -1,3 +1,10 @@
+- [[Plugins]] - general info on plugins
 - The draft plugins documentation is at https://logseq.github.io/plugins/, any feedback or suggestions are appreciated!
+
+## Some useful links
```

Comment:
These links are on the Plugins page so we should just focus on collecting them there

Reply:
On 2nd thought I like that you're collecting plugin dev links in this page. Will just remove embed for this page so it doesn't look weird